kanidmd/src/lib/be/dbvalue.rs

```diff
@@ -24,19 +32,28 @@ pub enum DbTotpAlgoV1 {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DbTotpV1 {
-    pub l: String,
-    pub k: Vec<u8>,
-    pub s: u64,
-    pub a: DbTotpAlgoV1,
+    #[serde(rename = "l")]
+    pub l: String, //? is this short for "label"?
+    #[serde(rename = "k")]
+    pub k: Vec<u8>, //? is this short for "secret"?
```

Comment:
key or secret

Reply:
I'm going to use `key` because `s` is already used for `step`.